src/tb/tb_style_edit.cpp

```diff
@@ -1262,7 +1264,11 @@ void TBTextFragment::Paint(int32_t translate_x, int32_t translate_y, TBTextProps
 		int line_h = font->GetHeight() / 16;
 		line_h = MAX(line_h, 1);
 		listener->DrawRectFill(TBRect(x, y + GetBaseline(font) + 1, GetWidth(font), line_h), color);
-	}
```

Comment:
Could text be both underlined and striken?

Reply:
There is no use case of combination of underlined/strikethrough/upperline.

Reply:
Not sure how to verify that?

iirc, it works in html?  What should a user expect to happen w/ nested tags?

https://www.w3schools.com/html/tryit.asp?filename=tryhtml_intro

Reply:
In css, it's expressed in css rule: [text-decoration](https://www.w3schools.com/cssref/tryit.php?filename=trycss_text-decoration). 

There's ``BBCode`` markup of  [Strikethrough](https://www.bbcode.org/making-strikethrough-text-with-bbcode.php).  [Godot Engine](https://docs.godotengine.org/en/stable/tutorials/ui/bbcode_in_richtextlabel.html) adopted the BBCode way.

 In my opinion, it is more like a convention, in actual world, there's seldom a need to combine these text-decorations. 





<br class="Apple-interchange-newline">